lib/xml/escape-attribute.js

```diff
@@ -0,0 +1,10 @@
+/**
+ * Escapes characters that can not be in an XML attribute.
+ */
+function escapeAttribute(value) {
+    return value.replace(/&/g, '&amp;').replace(/</g, '&lt;').replace(/>/g, '&gt;').replace(/"/g, '&quot;');
+}
```

Comment:
Do we need `&apos` for `'` here?

Reply:
We shouldn't need it because we always surround attributes with double quotes, but I can add it an rerun integration tests.